diff/fd/crosslaplacian.go

```diff
@@ -0,0 +1,190 @@
+// Copyright ©2017 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package fd
+
+import (
+	"math"
+	"sync"
+)
+
+// CrossLaplacian computes a Laplacian-like quantity for a function of two vectors
+// at the locations x and y.
+// It computes
+//  ∇_y · ∇_x f(x,y) = \sum_i ∂^2 f(x,y)/∂x_i ∂y_i
+// The two input vector lengths must be the same.
+//
+// Finite difference formula and other options are specified by settings. If
+// settings is nil, LaplacianTwo will be estimated using the Forward formula and
```

Comment:
Sorry, this operator doesn't actually have a name as best I can tell (but it does come up). I originally called it LaplacianTwo, but decided CrossLaplacian made more sense. 

Fixed.